functions/Clear-DbaSqlConnectionPool.ps1

```diff
@@ -44,31 +44,31 @@ function Clear-DbaSqlConnectionPool {
 		[Parameter(ValueFromPipeline = $true)]
 		[Alias("cn", "host", "Server")]
 		[DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
-		[PSCredential]$Credential,
+		[PSCredential][System.Management.Automation.Credential()]$Credential = [System.Management.Automation.PSCredential]::Empty,
 		[switch]$Silent
 	)
 
 	process
 	{
```

Comment:
If you want to update this in the function, the open curly brace should be on the same line as the starting command.

So it should be 
```
process {
```